src/data/modifiers.txt

```diff
@@ -5048,6 +5050,7 @@ Effect	Arse-a'fire	Muscle Percent: +10, Initiative: +25
 Effect	Artificially Sweet	Moxie Percent: +100
 Effect	Artisanal Retentive	Avatar: "Foodie Giant"
 Effect	Artisanal Satiaton	Experience: +3
+Effect	Ashamed	Experience: +10 ,Muscle:  -20%, Mysticality: -20%, Moxie: -20%
```

Comment:
```suggestion
Effect	Ashamed	Experience: +10, Muscle: -20%, Mysticality: -20%, Moxie: -20%
```

Reply:
Question: How did you generate this line?
Are you aware that you can generate these lines automatically in the correct formal?

test neweffect c9a095cc280c13e9ae6b44be77b61e79

puts the following in the gCLI - and in your session log, where you can copy it easily with actual tabs.
```
--------------------
2711	Ashamed	blush.gif	c9a095cc280c13e9ae6b44be77b61e79	neutral	none
Effect	Ashamed	Experience: +10, Muscle Percent: -20, Mysticality Percent: -20, Moxie Percent: -20
--------------------
```

Similarly, 

test newitem 836425101

does the same for items.
```
--------------------
10923	fire-roasted lake trout	836425101	fishy2.gif	food	t,d	10
fire-roasted lake trout	1	1	good	0	0	0	0	Unspaded
Item	fire-roasted lake trout	Effect: "Fire-Roasted Feelings", Effect Duration: 50
--------------------
--------------------
2715	Fire-Roasted Feelings	fishybone.gif	ecbc67d56ee9229d98ff596524e48cba	neutral	none	eat 1 fire-roasted lake trout
Effect	Fire-Roasted Feelings	Cold Resistance: +3, Hot Damage: +15
--------------------
```

Basically, if you know the item or effect descit, you can automatically generate everything you need in the formal you need it in. No manual entry and typos.

I am "unresolving" this, only so you can see it easily. I am not requiring an additional change. :)

Reply:
I had tried examining the effect, as I have found works with items, but came up with nothing. Thanks for sharing that command!

I also...definitely should have thought to check the session log instead of copy-pasting from the CLI. Oh well, you live and you learn.

Thanks!